src/main/java/com/github/dockerjava/api/model/SELContext.java

```diff
@@ -0,0 +1,44 @@
+package com.github.dockerjava.api.model;
+
+/**
+ * The context mode of bound volume in SELinux.
+ * Host path <code>shared</code> - can be used by all containers, <code>private</code> - by only this container
+ *
+ * @see http://www.projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/
+ */
+public enum SELContext {
```

Comment:
But you can just wrap it into `new SELContext()` instead of calling fromString. While library has no porcelain abstraction the idea is to keep it light and extensible between docker API and library usage.
I will try experiment myself my comments.